docs/src/modules/components/MarkdownElement.js

```diff
@@ -77,24 +77,22 @@ const markedOptions = {
   highlight(code, lang) {
     let language;
     switch (lang) {
-      case 'diff':
-        language = prism.languages.diff;
-        break;
-
-      case 'css':
-        language = prism.languages.css;
-        break;
-
       case 'ts':
-      case 'tsx':
         language = prism.languages.tsx;
         break;
 
       case 'js':
-      case 'jsx':
-      default:
+      case 'sh':
         language = prism.languages.jsx;
```

Comment:
That doesn't seem right. Isn't `sh` an alias for `bash`?

Reply:
Yes, it sounds like an alias: https://askubuntu.com/questions/172481/is-bash-scripting-the-same-as-shell-scripting.

Reply:
This code change doesn't change the logic.